packages/components/src/components/Text/index.tsx

```diff
@@ -12,11 +12,13 @@ export const Text = styled(Element).attrs({ as: 'span' })<{
   size?: number;
   align?: string;
   weight?: string;
+  block?: boolean;
   variant?: 'body' | 'muted' | 'danger';
-}>(({ size, align, weight, variant = 'body', ...props }) =>
+}>(({ size, align, weight, block, variant = 'body', ...props }) =>
   css({
     fontSize: size || 'inherit', // from theme.fontSizes
     fontWeight: weight || null, // from theme.fontWeights
+    display: block ? 'block' : 'inline',
```

Comment:
Found myself making <Text> elements block quite often so made a prop out of that. 

To add margin to a <Text> element, you would now do `<Text block marginBottom={2}>`